modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/RelatedObjectEntryOpenAPIContributor.java

```diff
@@ -187,6 +193,59 @@ private String _getContentType(
 		return dtoConverter.getContentType();
 	}
 
+	private Operation _getDeleteOperation(
+		ObjectDefinition objectDefinition,
+		ObjectRelationship objectRelationship,
+		SystemObjectDefinitionMetadata systemObjectDefinitionMetadata) {
+
+		String upperCaseFirstLetterObjectRelationshipName =
+			StringUtil.upperCaseFirstLetter(objectRelationship.getName());
+
+		DTOConverter<?, ?> dtoConverter = _dtoConverterRegistry.getDTOConverter(
+			systemObjectDefinitionMetadata.getModelClassName());
+
+		return new Operation() {
+			{
+				operationId(
+					"delete" + upperCaseFirstLetterObjectRelationshipName);
```

Comment:
This will generate an `operationId` like: `deleteStudentSubject` but the pattern that we follow says that it would generate something like `deleteUserAccountStudentSubject`

The other day, @ccorreagg realized that the other endpoints (GET and PUT) don't follow this pattern either and we have to change it. I don't know if Carlos created a ticket for that or took care of it in one of his PRs.

Reply:
I did not create any ticket, I included some modifications in my pull request. However, I think we should discuss which would be a proper operationId for all those cases

Reply:
I would go with the pattern used by the restBuilder for consistency 🤷‍♂️ 
We can discuss it

Reply:
Right! I didn't realize that we used a pattern for that :smile:. It will be great to disccuss it